BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBActivity.m

```diff
@@ -31,15 +31,43 @@
 //
 
 #import "_SBBActivity.h"
+#import "ModelObjectInternal.h"
 #import "NSDate+SBBAdditions.h"
 
+#import "SBBSurveyReference.h"
+#import "SBBSurveyResponseReference.h"
+#import "SBBTaskReference.h"
+
 @interface _SBBActivity()
 
 @end
 
+// see xcdoc://?url=developer.apple.com/library/etc/redirect/xcode/ios/602958/documentation/Cocoa/Conceptual/CoreData/Articles/cdAccessorMethods.html
+@interface NSManagedObject (Activity)
+
+@property (nullable, nonatomic, retain) NSString* activityType;
```

Comment:
this one (and the guid one below) are generated by the template to match what gets generated if you choose Create NSManagedObject Subclass... from the Editor menu when you select an entity in the model editor. Since these declarations are internal to the machine-generated files it shouldn't matter (and I think there are intermediate states e.g. while creating a new instance where the nascent object might not have these values yet, so I don't think I want to make them non-optional in the model, which doesn't seem to affect the generated code anyway)